cli/commands/configure.py

```diff
@@ -27,11 +27,9 @@ def configure(ctx: CliContext,
               output_format: Optional[str]) -> None:
     profile = ctx.confman.conf.active_profile
 
-    token = token or profile.token
```

Comment:
What is a profile without its token? That is the the only non optional filed of the profile.
Maybe I miss understood, lets talk about it.